lib/strings.md

```diff
@@ -5,20 +5,25 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A String is a sequence of characters either as literal constant or some kind of variable. 
+Strings are one of the most used data types in programming and are surrounded with single or double quotes. 
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+An example of strings is sentences or paragraphs on a website. Strings normally contain information to be read by a user. 
+Numbers for math related problems such as multiplication or adding would be used as integer/fixnum data type instead of strings. 
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+To find the 6th character it would be “Ada Lovelace” [6] and would be “v”.
+To find the 8th character it would be “Ada Lovelace”[8] and would be “l”.
```

Comment:
These explanations make sense to me if you are thinking of the `"A"` as the 0th letter in the string!